notebooks/source/_static/css/pyro.css

```diff
@@ -26,4 +26,8 @@
 
 .wy-side-nav-search>a img.logo, .wy-side-nav-search .wy-dropdown>a img.logo {
     max-width: 60%;
-}
\ No newline at end of file
+}
+
+.wy-nav-content {
+    max-width: 1000px !important;
```

Comment:
@jpchen - I am increasing the max width since I see many figures / tables which require horizontal scrolling. Should we also do this for Pyro?

Reply:
yeah that sounds reasonable